tasks/post-to-cmr/tests/cmr_test.js

```diff
@@ -80,6 +80,36 @@ test.serial('should succeed with correct payload', (t) => {
     }));
 });
 
+test.serial('should succeed with correct payload and return SIT url', (t) => {
+  process.env.CMR_ENVIRONMENT = 'SIT';
+  const newPayload = JSON.parse(JSON.stringify(payload));
+  sinon.stub(cmrjs.CMR.prototype, 'ingestGranule').callsFake(() => ({
+    result
+  }));
+  const granuleId = newPayload.input.granules[0].granuleId;
+  const key = `${granuleId}.cmr.xml`;
+
+  return aws.promiseS3Upload({
+    Bucket: t.context.bucket,
+    Key: key,
+    Body: fs.createReadStream('tests/data/meta.xml')
+  }).then(() => postToCMR(newPayload)
+    .then((output) => {
+      cmrjs.CMR.prototype.ingestGranule.restore();
```

Comment:
This line and the `delete ...` line after it could go in a `.finally()` block, since they're done on both success and failure.